X/XRootD/build_tarballs.jl

```diff
@@ -0,0 +1,103 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "XRootD"
+version = v"5.4.3"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/xrootd/xrootd/archive/refs/tags/v$(version).tar.gz", "56a29c88232f2f384e151b148fcaaa8d8db5c5fdc4615193978c8f4f3a99663c")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+mkdir build && cd build
+install_license ../xrootd-*/LICENSE
+cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release -DENABLE_PERL=FALSE ../xrootd-*/
+make -j${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = expand_cxxstring_abis(supported_platforms(; exclude=p->libc(p) != "glibc"))
```

Comment:
@giordano None-glibc platforms have been excluded here.

Reply:
I referred only to the depency libxcrypt_legacy_jll, not the entire recipe

Reply:
To clarify, none-glibc platforms were already excluded at first, not after applying lbxcrypt_legacy_jll.